alertwildfire_live_recording.py

```diff
@@ -54,8 +55,9 @@ def build_name_with_metadata(image_base_name,metadata):
 
     cameraName_chunk = image_base_name[:-23]
     metadata_chunk = 'p'+str(metadata['position']['pan'])+'_t'+str(metadata['position']['tilt'])+'_z'+str(metadata['position']['zoom'])
-    timeStamp_chunk = '__'+image_base_name[-23:]
-    imgname = cameraName_chunk+metadata_chunk+timeStamp_chunk
+    timeStamp_chunk = image_base_name[-23:-4]+'__'
+    fileTag=image_base_name[-4:]
+    imgname = cameraName_chunk+timeStamp_chunk+metadata_chunk+fileTag
```

Comment:
This is OK for now, but in future changes, we should switch to img_archive.parseFilename() and repack() for this type of formatting work.

Reply:
need to update parseFilename() and repack to handle new format of name